src/consts/pipelinedata.ts

```diff
@@ -0,0 +1,25 @@
+export const PIPELINE_DATA = {
```

Comment:
This data would only be used by useBuildPipelineConfig in case ConfigMap was not found or request resulted in an error. So I think it's safe to keep this in the same file `useBuildPipelineConfig`